nearlib/dev.js

```diff
@@ -9,16 +9,23 @@ module.exports = {
     connect: async function(nodeUrl) {
         const studioConfig = await this.getConfig();
         const near = nearlib.Near.createDefaultConfig(nodeUrl || studioConfig.nodeUrl);
-        await this.getOrCreateDevUser();
+        await this.getOrCreateDevUser(near);
         return near;
-    }, 
-    getOrCreateDevUser: async function () {
+    },
+    getOrCreateDevUser: async function (near) {
```

Comment:
I think we should either move this method to `near.dev` or keep default dev instance of `near` in `nearlib.dev`. WDYT?

Reply:
yeah this method makes more sense in dev

Reply:
wait it's already in dev, isn't it?

Reply:
it's in `nearlib.dev`. `nearlib.dev` doesn't have connection to node (so you currently pass `near` as param).

Reply:
This is finicky to change/test. I prefer to do it in next iteration to avoid last minute bug fixes tomorrow